official/transformer/utils/schedule_test.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
```

Comment:
Note to selves: BUILD files required. Though I don't think we have any for Transformer yet, so just whenever those are created @k-w-w

Reply:
I would like to get the copybara changes in before creating the BUILD files (so the tests don't fail from the start). Are they causing merges to be blocked?